Tasks/SonarQubePostTest/PRCA/Orchestrator.ps1

```diff
@@ -77,7 +77,7 @@ function GetMessagesFromIssues
         [void]$comments.Add($comment)
     }
     
-    Write-Verbose "Creating $($issues.Count) messages from issues took $($sw.ElapsedMilliseconds) ms"
+    Write-VstsTaskVerbose "Creating $($issues.Count) messages from issues took $($sw.ElapsedMilliseconds) ms"
```

Comment:
It's a common practice to stop the stop watch before reading the time... (even though either work)